docs/content/en/concepts/logical/designs.md

```
@@ -9,6 +9,55 @@ Like a Google Doc, Designs are your primary tool for collaborative authorship of
 
 As the deployable unit in Meshery, a Design consists of [Components]({{< ref "concepts/logical/components.md" >}}) and [Relationships]({{< ref "concepts/logical/relationships/index.md" >}}). Designs are how you can describe your desired infrastructure state.
 
+## Types of Meshery Designs
+
+Meshery supports two primary classifications of designs to accommodate both architectural planning and infrastructure orchestration:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)